apps/condo/domains/resident/schema/RegisterResidentService.test.js

```diff
@@ -431,6 +431,24 @@ describe('RegisterResidentService', () => {
         expect(restoredResident.property.id).toEqual(property.id)
     })
 
+
+    it('restore deleted Resident for the same address and unitName updates unitName', async () => {
```

Comment:
Isnt it DOMA-9474?

Reply:
They look like duplicates, highlighting ability to change unitName to be able to transmit the readings, but 9475 has more info

Reply:
They are not duplicates, they are different 

74 is about fixing deletion/restore
75 is about fixing Resident.organizationFeatures